_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,308 @@
+---
+id: c249d5
+name: Device motion can be disabled
+rule_type: atomic
+description: |
+  This rule checks that it is possible to disable functionality that can be operated by device motion.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
+
+## Expectation 1
+
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] is available to disable the event.
+
+**Note:** The same [instrument][] can be used to disable more than one event.
+
+## Expectation 2
+
+The [instrument][] is [visible][].
+
+## Expectation 3
+
+The [instrument][] is [included in the accessibility tree][] with an [accessible name][] that is not empty ("").
+
+## Assumptions
+
+- The motion to operate the device is not used through an [accessibility supported][] interface.
```

Comment:
I don't really understand what this means. Can you make this clearer?

Reply:
It's one of the exceptions in the success criterion. I'll make that clear.